fastlane/spec/actions_specs/xcodebuild_spec.rb

```diff
@@ -27,6 +27,7 @@
             destination_timeout: 240,
             dry_run: true,
             enable_address_sanitizer: true,
+            enable_thread_sanitizer: true,
```

Comment:
Agree. I'll just remove it from here, there are separate tests for address and thread sanitizer options anyways.